ee/benchmarks/benchmarks.py

```diff
@@ -458,6 +459,26 @@ def track_retention_filter_by_person_property_materialized(self):
 
         ClickhouseRetention().run(filter, self.team)
 
+    @benchmark_clickhouse
+    def track_lifecycle(self):
+        filter = Filter(
+            data={
+                "insight": "LIFECYCLE",
+                "events": [{"id": "$pageview", "name": "$pageview", "type": "events", "order": 0, "math": "total"}],
+                "display": "ActionsLineGraph",
+                "interval": "week",
+                "shown_as": "Lifecycle",
+                "date_from": "-14d",
+                "properties": [],
+                "compare": False,
+                "filter_test_accounts": True,
```

Comment:
Nit: you can remove this + compare + properties and you _should_ remove interval and date_from as they're overridden.